pkg/detector/preemption.go

```diff
@@ -227,3 +228,48 @@ func (d *ResourceDetector) fetchResourceTemplate(rs policyv1alpha1.ResourceSelec
 	}
 	return resourceTemplate, nil
 }
+
+// HandleDeprioritizedPropagationPolicy responses to priority change of a PropagationPolicy,
+// if the change is from high priority (e.g. 5) to low priority(e.g. 3), it will
+// check if there is another PropagationPolicy could preempt the targeted resource,
+// and put the PropagationPolicy in the queue to trigger preemption.
+func (d *ResourceDetector) HandleDeprioritizedPropagationPolicy(oldPolicy policyv1alpha1.PropagationPolicy, newPolicy policyv1alpha1.PropagationPolicy) {
+	if (newPolicy.Spec.Priority != nil && oldPolicy.Spec.Priority != nil) && *newPolicy.Spec.Priority < *oldPolicy.Spec.Priority {
```

Comment:
use `newPolicy.ExplicitPriority() <= oldPolicy.ExplicitPriority()` instead.

Reply:
+1, totally agree, thanks for the reminder.

Reply:
`<=` or `<`?
I guess it should be `<`.

Reply:
> I guess it should be <.

right, sorry, it's my mistake.